README.md

```diff
@@ -32,25 +32,13 @@ Don't forget to ⭐ this repo if you like immudb!
 
 ---
 
-immudb is a **lightweight, high-speed immutable database** with built-in cryptographic proof and verification.
-With immudb, you can track changes in sensitive data in your transactional databases and then record those changes permanently in a
-tamperproof immudb database.
-<img align="right" src="img/immudb-mascot-small.png" width="256px"/>
-
-Traditional DB transactions and logs are hard to scale and are mutable, so there is no way to know for sure if your data has been compromised.
-immudb provides **unparalleled insights** **retroactively** of changes to your sensitive data, even
-if your perimeter has been compromised. immudb guarantees immutability by using **cryptographic data structures** internally.
-This allows you to keep a persistent history of sensitive data, such as debit/credit card transactions.
+immudb is a key-value database with built-in cryptographic proof and verification. It can track changes in sensitive data and the integrity of the history will be protected by the clients, without the need to trust the server.
```

Comment:
"Traditional DB transactions and logs are hard to scale and are mutable, so there is no way to know for sure if your data has been compromised."

This line is useful to explain the value proposition of immudb -- I'd keep it.

Reply:
I think this "so there is no way to know for sure if your data has been compromised." is covered by the next part: "This lets you store critical data without fear of it being changed silently"

Reply:
I'd still keep it -- the latter is about immudb, the former is about traditional databases. Having both clauses sets up a compare-and-contrast for the reader.